### DISCUSSION TITLE
Populate "mentions" tab with recent messages

### DISCUSSION THREAD

**Discussion by 2547techno** [Chatterino/chatterino2#4474] [Ideas]
Currently highlighting options are applied to messages from recent-messages but are not added to the mentions tab.

The suggestion is to add this functionality.

EDIT: The intended purpose is to quickly see mentions on first load without the need of checking each tab for scrollbar highlights

**Comment by Felanbird** [comment]
More or less discussed in #2572

**Reply by 2547techno** [reply]
The idea was so that when you open chatterino for the day/week/etc you don't have to scroll through your tabs and look for scrollbar highlights, instead just needing to look at your mentions tabs. The caching method is for a slightly different problem. I've edited the post, thanks :)

**Reply by iProdigy** [reply]
In the meantime, you can use Ctrl+Shift+F to search for a specific string across your tabs

**Reply by Felanbird** [reply]
> The idea was so that when you open chatterino for the day/week/etc you don't have to scroll through your tabs and look for scrollbar highlights, instead just needing to look at your mentions tabs. 

Yes I'm aware, which is exactly what gempir suggests in that issue.

**Comment by iProdigy** [comment]
Questions:
* should populating mentions from recent messages only apply to initial app start, or whenever a user joins a new channel in general?
* Should historical mentions be appended to the bottom of the mentions split or should it be inserted according to the timestamp to maintain chronological order?

**Reply by 2547techno** [reply]
~~I would say on join & chronological~~ maybe app start & chronological makes more sense

**Comment by 2547techno** [comment]
#4475 an implementation
